PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -11,8 +11,8 @@
         'keep *_slimmedTausBoosted_*_*',
         'keep *_slimmedCaloJets_*_*',
         'keep *_slimmedJets_*_*',
-        # drop content created by MINIAOD DeepFlavour production
-        'drop recoBaseTagInfosOwned_slimmedJets_*_*',
+        # keep slimmedJets TagInfos, currently only PixelClusterTagInfo
+        'keep recoBaseTagInfosOwned_slimmedJets_*_*',
```

Comment:
This line is redundant, as it is fully contained in L13, but maybe it can be left as a reminder for the presence of those TagInfos.
@ferencek please anyhow have a look and confirm whether this satisfies your comment.

Reply:
Thinks look ok to me now. Indeed, L15 is redundant but I agree it probably doesn't hurt to leave it along with L14 as a reminder.